src/App.tsx

```diff
@@ -17,18 +61,29 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterBy={filterBy}
+                setFilterBy={setFilterBy}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length
```

Comment:
More correact to add additional isLoading state variable, to indicate todos loading. It's need for prevent infinity loader, if todos list is empty